code.gs

```diff
@@ -550,7 +550,7 @@ function checkEvents (testDate) {
     body = [bodyPrefix, '\n']
         .concat(bodyBuilder)
         .concat(['\n\n ', bodySuffix1, '\n ', bodySuffix2, '\n'])
-        .concat('\n', isRunningOutdatedVersion() ? [bodySuffix3, ' ', _('here'), ':\n', baseGitHubProjectURL + 'releases/latest', '\n '] : [])
+        .concat('\n', isRunningOutdatedVersion() ? [bodySuffix3, '\n', baseGitHubProjectURL + 'releases/latest', '\n '] : [])
```

Comment:
Oops, did it again sorry. I only use plaintext email so I forget to check html output when doing stuff in a hurry. It seems a bit silly to suggest unit tests for a single-file script, but then again...